scene/main/viewport.cpp

```diff
@@ -1265,7 +1265,12 @@ void Viewport::_gui_show_tooltip() {
 	Rect2 r(gui.tooltip_pos + tooltip_offset, gui.tooltip_popup->get_contents_minimum_size());
 
 	Window *window = gui.tooltip_popup->get_parent_visible_window();
-	Rect2i vr = window->get_usable_parent_rect();
+	Rect2i vr;
+	if (gui.tooltip_popup->is_embedded()) {
+		vr = Rect2i(Point2i(), gui.tooltip_popup->_get_embedder()->size);
```

Comment:
I just noticed, that `size` is not enough.
As soon, as the window is scaled (`Window.content_scale_mode = CONTENT_SCALE_MODE_CANVAS_ITEMS`), it becomes necessary to use
```
    vr = gui.tooltip_popup->_get_embedder()->get_visible_rect();
```
because `size` does not necessarily indicate the visible area of the embedder, which again leads to mispositioned windows.